### PR TITLE
feat: cluster health report

### PR DESCRIPTION
**Pull request by kamciokodzi**
https://github.com/fishjam-dev/fishjam/pull/213

### CODE REVIEW

File: poetry_scripts.py [hunk 1]
```diff
@@ -66,8 +66,8 @@ def generate_docs():
     check_exit_code(
         "pdoc \
     --include-undocumented \
-    --favicon https://logo.swmansion.com/membrane/\?width\=100\&variant\=signetDark\
-    --logo https://logo.swmansion.com/membrane/\?width\=70\&variant\=signetDark\
+    --favicon https://logo.swmansion.com/membrane/?width=100&variant=signetDark\
+    --logo https://logo.swmansion.com/membrane/?width=70&variant=signetDark\
```
Comment (sgfn) [inline poetry_scripts.py]: Are you sure un-escaping the ampersand `&` here doesn't cause the command to break?

File: poetry_scripts.py [hunk 2]
```diff
@@ -90,7 +90,7 @@ def update_client():
     check_exit_code(
         "openapi-python-client update\
             --url https://raw.githubusercontent.com/fishjam-dev/"
-        "fishjam/main/openapi.yaml \
+        "fishjam/feat/cluster-health-report/openapi.yaml \
             --config openapi-python-client-config.yaml \
```
Comment (sgfn) [inline poetry_scripts.py]: We should keep using `main` here and only generate the SDKs after the relevant PR in `fishjam` gets merged

File: poetry_scripts.py [hunk 3]
```diff
@@ -90,7 +90,7 @@ def update_client():
     check_exit_code(
         "openapi-python-client update\
             --url https://raw.githubusercontent.com/fishjam-dev/"
-        "fishjam/main/openapi.yaml \
+        "fishjam/feat/cluster-health-report/openapi.yaml \
```
Comment (Rados13) [inline poetry_scripts.py]: I think we don't want to change this line.

**Review by Rados13 — COMMENTED** [review]
What do you think about testing this changed functionality?

File: poetry_scripts.py (hunk 1 above)
Comment (kamciokodzi) [reply poetry_scripts.py]: Ah sorry for those changes I forgot to untrack this file.

If you are using python3 all of those escaping is unnecessary and throws an error.

File: poetry_scripts.py (hunk 3 above)
Comment (kamciokodzi) [reply poetry_scripts.py]: Ah sorry for those changes I forgot to untrack this file.

**Pull request closed without merging**